README.md

```diff
@@ -7,9 +7,52 @@ Covata Delta provides an easy to use framework for sharing secrets across networ
 
 ## Requirements
 
-- Python 2.7 +
-- Python 3.3 +
+- Python 2.7 + or 3.3 +
+- pip 9.0.1 +
+- virtualenv
+- pybuilder
 
+## Quick Start
+
+### Setting up Virtualenv and PyBuilder
+
+```bash
+sudo pip install virtualenv
+virtualenv venv
+source venv/bin/activate
+pip install pybuilder 
+```
+
```

Comment:
Add a comment to say you can now either build the project from source, or do a pip install via github.